CHANGES.rst

```diff
@@ -69,7 +69,7 @@ Major changes
 
 * Improvements to the :class:`MinHashEncoder`
     - It is now possible to fit multiple columns simultaneously with the :class:`MinHashEncoder`.
-    Very useful when using for instance the `~sklearn.compose.make_column_transformer` method,
+    Very useful when using for instance the :class:`~sklearn.compose.ColumnTransformer` method,
```

Comment:
The original was correct, it is in fact `make_column_transform` :)

Reply:
Oh thanks, that would be a big miss :)

Reply:
[The link doesn't work](https://output.circle-artifacts.com/output/job/54395ea9-4d05-41e3-b497-40d7617e1c38/artifacts/0/doc/CHANGES.html), I guess it's because it's not a class, but a function :)

Reply:
Thanks, should work now!